File: CS/Scaffolding.CustomFilter/IssueContextDataModel/Model/IIssueRepository.cs
using System;
using System.Linq;
using System.Data;
using System.Data.Entity;
using System.Linq.Expressions;
using System.Collections.Generic;
using Scaffolding.CustomFilter.Common.Utils;
using Scaffolding.CustomFilter.Model;
using Scaffolding.CustomFilter.Common.DataModel;
using Scaffolding.CustomFilter.Common.DataModel.EntityFramework;

namespace Scaffolding.CustomFilter.IssueContextDataModel {
    public interface IIssueRepository : IRepository<Issue, int> {
    }
}

File: VB/Scaffolding.CustomFilter/IssueContextDataModel/Model/IIssueRepository.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Data
Imports System.Data.Entity
Imports System.Linq.Expressions
Imports System.Collections.Generic
Imports Scaffolding.CustomFilter.Common.Utils
Imports Scaffolding.CustomFilter.Model
Imports Scaffolding.CustomFilter.Common.DataModel
Imports Scaffolding.CustomFilter.Common.DataModel.EntityFramework

Namespace Scaffolding.CustomFilter.IssueContextDataModel
	Public Interface IIssueRepository
	Inherits IRepository(Of Issue, Integer)
	End Interface
End Namespace